ci/src/main/scala/org/typelevel/sbt/CrossRootProject.scala

```diff
@@ -126,8 +128,10 @@ object TypelevelCiJVMPlugin extends AutoPlugin {
   override def requires = TypelevelCiCrossPlugin
 
   override def buildSettings: Seq[Setting[_]] = Seq(
-    githubWorkflowBuildMatrixAdditions ~= { matrix =>
-      matrix.updated("project", matrix("project") ::: "rootJVM" :: Nil)
+    githubWorkflowBuildMatrixAdditions := {
+      val matrix = githubWorkflowBuildMatrixAdditions.value
+      val rootProject = (LocalRootProject / Keys.thisProject).value
```

Comment:
To save some repetition, we could extract `rootProjectId` into a `Def.setting` maybe.

Reply:
Hmm I thought about doing so, but once you have declared a SettingKey in your plugin it kind of becomes part of the public API of that plugin across every scope axis - do you think it's worth it? Totally happy to do so if you prefer that!

Reply:
Hmm, I might be learning something here, but isn't it possible to have anonymous settings? I hope so because I used them everywhere 😆 For example like this one, what's the setting key for it?
https://github.com/typelevel/sbt-typelevel/blob/50e6ddfb93ff143c293087d5b8159738a0f3363f/github/src/main/scala/org/typelevel/sbt/TypelevelGitHubPlugin.scala#L58

Reply:
Oh no it's definitely me that's learning - that's a nice pattern that I hadn't seen before

Reply:
OK applied that one as well now 🚀